examples/cifar10/pt/learners/cifar10_learner.py

```diff
@@ -51,6 +53,7 @@ def __init__(
             aggregation_epochs: the number of training epochs for a round. Defaults to 1.
             train_task_name: name of the task to train the model.
             submit_model_task_name: name of the task to submit the best local model.
+            stream_tb: whether to send TensorBoard values to server. Defaults to False.
```

Comment:
Suggest to change: `whether to send TensorBoard values. Defaults to False.`

Because the TB event may not be sent to the server.
It is sent to the server here only because we are using the ConvertToFedEvent widget.